internal/controller/resources/template/MULTI_CLUSTER_AUTH.md

```
@@ -0,0 +1,1445 @@
+Generate a key pair:
+
+```bash
+openssl ecparam -name prime256v1 -genkey -noout -out key.pem
+openssl ec -in key.pem -pubout -out pub.pem
+```
+
+## Wristband issuer URL
+
+The `issuer` field follows the format:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)